fix: remove nms-hybrid keyword (#2044)
nginx/documentation: content/nim/deploy/kubernetes/frequently-used-helm-configs.md

You can review and adjust the deployment's default resource and storage settings by editing the `values.yaml` file in the Helm package you downloaded. Adjust the values to meet your data needs.

> Persistent volumes are enabled by default for the ClickHouse database server and the `core` and `dpm` services. To disable persistent storage for a configuration, set `<service>.persistence.enabled` to `false`.

Reviewer: ```suggestion
Persistent volumes are on by default for the ClickHouse database server and the `core` and `dpm` services. To turn off persistent storage, set `<service>.persistence.enabled` to `false`.
```

Edit for style. See: https://github.com/F5Docs/style-guide/blob/main/terminology/enable-disable.md
